.github/workflows/stale.yml

```diff
@@ -1,22 +1,24 @@
+---
 # This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
 #
 # You can adjust the behavior by modifying this file.
 # For more information, see:
 # https://github.com/actions/stale
 
-name: 'Close stale issues and PRs'
-on:
+name: Close stale issues and PRs
+true:
```

Comment:
This is not right and breaks the workflow

Reply:
Outch. Mabe we can fix this with one of the next pull requests.